source/configuration_manual/mail_crypt_plugin.rst

```diff
@@ -331,7 +331,8 @@ global key can be provided with several different scopes:
 
 With folder keys, key sharing can be done to single user, or multiple users.
 When key is shared to single user, and the user has public key available, the
-folder key is encrypted to recipient's public key.
+folder key is encrypted to recipient's public key. This requires `mail_crypt_acl` plugin,
+which will enable accessing encrypted shared folders.
```

Comment:
Three comments:
1. I would appreciate it, if there were more articles: "[...] key sharing can be done to a single user [...]" and "When _a_ key is shared [...], has _a_ public key [...] to _the_ recipient's public key."; "This requires _the_ `mail_crypt_acl."
2. I have not found a reference to the "mail_crypt_acl" plugin anywhere, only to "mail_crypt_acl_require_secure_key_sharing".
3. The line exceeds 80 chars.

Reply:
yeah, i'm kinda adding the reference here since we forgot to document it.

Reply:
also the 80 character limit is soft limit, it does not need to be stricly enforced.